css/main.css

```diff
@@ -325,3 +325,62 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 24px;
+}
+
+.footer-text {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-form-container {
+  width: 100%;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-form .contact-form-input {
+  width: 100%;
+  height: 48px;
+  font-size: 15px;
+  line-height: 24px;
+  border: none;
+  outline: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 0 16px;
+  margin-bottom: 20px;
+}
+
+.contact-form .message {
+  background: #fbf8f7;
+  width: 100%;
+  padding: 12px;
+  font-size: 15px;
+  line-height: 24px;
+  margin-bottom: 20px;
+  outline: none;
+}
+
+.contact-submit {
+  height: 48px;
+  width: 131px;
+  background-color: #000;
+  color: #fff;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  border: none;
+}
```

Comment:
- [ ] Please make sure that the colour of the submit button respects the figma template.

- [ ]  At the same time, we need to ensure that on hover, the colour of the button changes. 

Here is the design from figma regarding the colour of buttons:
![image](https://user-images.githubusercontent.com/83596135/152647963-7dfe2ac6-eb56-4389-a6c8-b63e91124197.png)

Reply:
- As the previous code reviewer said, the submit button should look exactly like the button in the Figma design. 

- [ ] Here is your button :
![image](https://user-images.githubusercontent.com/71874425/152653888-70b1eddc-26ff-46a9-8490-90a16967d7e6.png)
Kindly, please change the color, font, and background color of your submit button and also implement the button interactions(enable, hover) so that it looks like the submit button from the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).